engine/src/engine.rs

```diff
@@ -1049,9 +1051,17 @@ impl ApplyBackend for Engine {
                     let generation = Self::get_generation(&address);
                     Engine::set_nonce(&address, &basic.nonce);
                     Engine::set_balance(&address, &Wei::new(basic.balance));
+                    writes_counter += 2; // 1 for nonce, 1 for balance
 
                     if let Some(code) = code {
-                        Engine::set_code(&address, &code)
+                        Engine::set_code(&address, &code);
+                        code_bytes_written = code.len();
+                        sdk::log!(crate::prelude::format!(
+                            "Address {:?} set code with {} bytes",
+                            address,
+                            code_bytes_written,
```

Comment:
Should be written as two logs such as:
`written_bytes <address> <code_bytes_writte` that was its easier for a tool to parse.